engine/src/main/java/org/terasology/world/block/entity/placement/BlockPlacingSystem.java

```diff
@@ -34,6 +38,9 @@
 
     @ReceiveEvent(components = {WorldComponent.class}, priority = EventPriority.PRIORITY_TRIVIAL)
     public void placeBlockInWorld(PlaceBlocks event, EntityRef world) {
-        worldProvider.setBlocks(event.getBlocks());
+        //   worldProvider.setBlocks(event.getBlocks());
```

Comment:
do we still need this? if so why is it commented out? if not, why is it still there :yum:

Reply:
looks like the same line but commented out :P

Reply:
opps